neurokit2/bio/bio_analyze.py

```diff
@@ -35,6 +35,17 @@ def bio_analyze(data, sampling_rate=1000, method="auto", window_lengths='constan
     window_lengths : dict
         Defaults to 'constant'. Add a dictionary of epoch start and end times for different
         types of signals e.g., window_lengths = {'ECG': [0.5, 1.5], 'EDA': [0.5, 3.5]}
+    time_start : int, dict
+        For event-related analysis, a smaller epoch within the epoch of an event can be specified.
```

Comment:
add in the first sentence something like "and are currently only used for the rate-related features (min, max, etc.)"

Reply:
hmm but the sentence after that is "The ECG and RSP rate-related features of this "sub-epoch" (e.g., ECG_Rate, ECG_Rate_Max),  relative to the baseline (where applicable), will be computed". Should I move it to the first sentence?

Reply:
yeah and/or change the arg name to the more explicit one?

Reply:
yea..changing to rate_start and rate_end like you suggested